### DISCUSSION TITLE
Back, Cancel and Confirm Buttons too small

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#417] [Q&A] [answered]
I have recognized that these buttons in the menus including action menus are too small to be comfortably used with motorcycle gloves.
Can you please make them larger, especially in height (in portrait and landscape format)? I have problems getting them right, especially when the menus are at the edge and the space for operation is further restricted by navigation brackets. They should also be framed with a different colour than the other buttons so that they are easier to see while driving, possibly with a different background colour than the other buttons.

Thanks in advance.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the suggestion.

I can see what we can do. :slightly_smiling_face: 

Note: especially in landscape, a larger size (of anything) can cause a scroll bar on elements.

**Comment by devemux86** [comment, marked as answer]
We can make those buttons a little bigger and see how it works.
The touch area of the buttons is anyway larger than the icons.

About their appearance, if they are filled, they may look more bothersome to the eye and less recognizable:

<img src="https://github.com/devemux86/cruiser/assets/3484020/1ec49c55-c13e-4f70-9ce6-d9bbb0be6339" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/978f6697-21e5-4f11-9781-0ccdbfb796df" width="50%"/>

**Reply by umpfmpf** [reply]
In all menus the x is on the lower right side, only in the profile menu it is moved. Perhaps you can also slide the hook into the middle and the x to the right. 
And a optical fume off around the button could also be an optical upgrade.

**Reply by devemux86** [reply]
@umpfmpf the positive action is usually on the right, this is also the Android default.

Alternatively we could show the check mark in the dialogs that now only have the X.
When we had the words, the word "OK" was only displayed.

**Reply by umpfmpf** [reply]
I think like colored buttons with a frame around.
![Screenshot_20240107_212947_Cruiser](https://github.com/devemux86/cruiser/assets/105640513/d1b1d946-3fe7-4901-87f5-f22d426312c2)
![photostudio_1704660261498](https://github.com/devemux86/cruiser/assets/105640513/dfdd2d5c-c209-4d52-82c2-4f13309cc0bd)

**Reply by airald** [reply]
left picture proposed by @devemux86 is more recognizable as the entire symbol is different. The right ones have more in common between them.

**Reply by Ciutnik** [reply]
I also like the left image better, but maybe an outline of the touch area would actually be useful. Maybe highlight buttons like in the main menu?
